src/main/java/cubicchunks/server/CubeProviderServer.java

```diff
@@ -187,6 +188,11 @@ public boolean saveChunks(boolean alwaysTrue) {
     public boolean tick() {
         // NOTE: the return value is completely ignored
         // NO-OP, This is called by WorldServer's tick() method every tick
```

Comment:
You should probably change that. Then it should be ok to merge for now. If I want any further changes, I can do it myself some time later.